### ISSUE TITLE
Make a recommendation on which license to use to publish tilt packages

### ISSUE THREAD

**Issue by Tilmon** [2DegreesInvesting/tiltIndicator#659]

**Comment by maurolepore** [comment]
I use advise from [here](https://r-pkgs.org/license.html):

* If you want a permissive license so people can use your code with minimal restrictions, choose the [MIT license](https://choosealicense.com/licenses/mit/) with use_mit_license().
* If you want a copyleft license so that all derivatives and bundles of your code are also open source, choose the [GPLv3 license](https://choosealicense.com/licenses/gpl-3.0/) with use_gpl_license().
* If your package primarily contains data, not code, and you want minimal restrictions, choose the [CC0 license](https://choosealicense.com/licenses/cc0-1.0/) with use_cc0_license(). Or if you want to require attribution when your data is used, choose the [CC BY license](https://choosealicense.com/licenses/cc-by-4.0/) by calling use_ccby_license().
* If you don’t want to make your code open source, call use_proprietary_license(). Such packages can not be distributed by CRAN.

---

We went through this conversation with the r2dii packages. We started with GLP but we then changed to MIT. See them here:

* https://cloud.r-project.org/web/packages/r2dii.match/index.html
* https://cloud.r-project.org/web/packages/r2dii.analysis/index.html
* https://cloud.r-project.org/web/packages/r2dii.plot/index.html

**Comment by Tilmon** [comment]
Thanks @maurolepore, that helps a lot! Why did you move from GPL to MIT with PACTA?

**Comment by maurolepore** [comment]
To give users total freedom. The product owners of PACTA didn' mind users potentially selling a product powered by our open source software.

**Comment by Tilmon** [comment]
Thanks! 
Note to myself for future reference that under the link that you, @maurolepore , shared, there is a link to the author's ([Mara Averick](https://twitter.com/dataandme)) notes on re-licensing and things to consider when making decisions around licenses which are super useful for us I think: ["Notes on re-licensing" ](https://colorado.posit.co/rsc/relicensing-the-notes/the-notes.html)
cc' @AnneSchoenauer

**Comment by AnneSchoenauer** [comment]
@Tilmon did you write a ticket on changing liceneses based on our decision?

**Comment by Tilmon** [comment]
yep, @AnneSchoenauer please see here Change package licenses to GPLv3
[#670](https://github.com/2DegreesInvesting/tiltIndicator/issues/670)
